Decimus/LOCRegistry.swift

```diff
@@ -0,0 +1,242 @@
+// SPDX-FileCopyrightText: Copyright (c) 2025 Cisco Systems
+// SPDX-License-Identifier: BSD-2-Clause
+
+import CoreMedia
+import Foundation
+
+// Dictionary of header extension keys to values
+typealias HeaderExtensions = [NSNumber: Data]
+
+extension HeaderExtensions {
+    mutating func setHeader(_ mediaExtension: MediaTypeHeaderExtension) throws {
```

Comment:
Can we create 2 versions 
1. SetMoQMiHeaderXXXX
2. SetLoCHeaderXXXX

first of it will be called only when the toggle is set.